src/modeler/AutoRest.Swagger/Validation/OperationsAPIImplementationValidation.cs

```diff
@@ -40,6 +40,11 @@ public class OperationsAPIImplementationValidation : TypedRule<Dictionary<string
         public override Category Severity => Category.Error;
 
         /// <summary>
+        /// What kind of change implementing this rule can cause.
+        /// </summary>
+        public override ValidationChangesImpact ValidationChangesImpact => ValidationChangesImpact.NonBreakingServiceChanges;
```

Comment:
this is one where I'm not sure we want to categorize between breaking or non-breaking, maybe having "potential service impact" and "potential sdk impact" could work better and in that case, I think this would fall into "potential service impact" even though it may not be breaking.
@kirthik would you want this one to show up in reports as service impacting?